CHANGELOG.md

```diff
@@ -40,7 +40,7 @@ WIP:
 
 * gps: Process canonical import paths. (#1017)
 * gps: Persistent cache. (#1127, #1215)
-
+* dep: Enable importing other tools and add integration tests
```

Comment:
We just released v0.3.2. Please move this under v0.3.3.
And also add the PR #. 🙂

Reply:
Updated the PR with the changes.